lib/rpmsg/rpmsg_virtio.c

```diff
@@ -670,7 +678,21 @@ int rpmsg_virtio_get_buffer_size(struct rpmsg_device *rdev)
 		return RPMSG_ERR_PARAM;
 	metal_mutex_acquire(&rdev->lock);
 	rvdev = (struct rpmsg_virtio_device *)rdev;
-	size = _rpmsg_virtio_get_buffer_size(rvdev);
+	size = _rpmsg_virtio_get_buffer_size(rvdev, RPMSG_BUFFER_TX);
+	metal_mutex_release(&rdev->lock);
+	return size;
```

Comment:
Please add a blank line before the return

Reply:
Done